debian/k8s-cni/99_bridge.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "cniVersion": "0.1.0",
+  "name": "kubenet",
```

Comment:
This is not related to the kubelet kubenet plugin, (which doesn't just work on GCE, it works everywhere). This is the "name" field of the bridge cni plugin which represents the name of the network.

https://github.com/containernetworking/cni/blob/master/Documentation/bridge.md

The convenient way to support other cni plugins is add a config file that lexically sorts to a higher slot then 99_bridge.conf, e.g. 10_flannel.conf. Bridge plugin seems like a reasonable default